### ISSUE TITLE
Hanging indent for multiple line line array entry

### ISSUE THREAD

**Issue by scottgrayson** [emacs-php/php-mode#623]
### Indented
```php
$arr = [
    $this->something()
    ->something(),
];

$this->something()
    ->something();
```
### expected
```php
$arr = [
    $this->something()
        ->something(),
];

$this->something()
    ->something();
```
### config
```org
  (use-package php-mode
    :ensure t
    :config
    (setq c-basic-offset 4)
    (add-hook 'php-mode-hook 'php-enable-psr2-coding-style)
    )
```

If the the result shown in `expected` cannot become the default, how can I customize this with config?

Another example of this https://nova.laravel.com/docs/3.0/resources/fields.html#keyvalue-field

**Comment by zonuexe** [comment]
@scottgrayson Thank you for reporting. Probably solved in #624.

**Comment by scottgrayson** [comment]
that solved it
